### ISSUE TITLE
Too many InvalidReferenceViolation messages

### ISSUE THREAD

**Issue by jamesaoverton** [ontodev/robot#374]
When reasoning over most of the ontologies that I build, I see a lot of these "ERROR" messages that are not genuine errors. Here's an example @zhengj2007 sent me:

```
ERROR org.obolibrary.robot.ReasonOperation - Reference violations found: 169 - reasoning may be incomplete
2018-10-04 11:20:28,093 ERROR org.obolibrary.robot.ReasonOperation - Reference InvalidReferenceViolation [axiom=SubClassOf(<http://purl.obolibrary.org/obo/BFO_0000001> owl:Thing), referencedObject=owl:Thing, category=DANGLING]
```

I'd like to suppress these. Maybe a whitelist of terms to ignore would be enough. Maybe there's a more general solution. @cmungall ?

See also #1 and #259.

**Comment by mcourtot** [comment]
Seeing the same - shall I just ignore those?

`C02XG3Q1JHD3:ontology courtot$ make test
robot  reason -i duo-edit.owl -r ELK relax reduce -r ELK annotate -V http://purl.obolibrary.org/obo/duo/releases/`date +%Y-%m-%d`/duo.owl -o duo.owl
2018-12-18 11:21:35,447 ERROR org.obolibrary.robot.ReasonOperation - Reference violations found: 63 - reasoning may be incomplete
2018-12-18 11:21:35,448 ERROR org.obolibrary.robot.ReasonOperation - Reference violation: InvalidReferenceViolation [axiom=SubClassOf(<http://purl.obolibrary.org/obo/BFO_0000001> owl:Thing), referencedObject=owl:Thing, category=DANGLING]
2018-12-18 11:21:35,448 ERROR org.obolibrary.robot.ReasonOperation - Reference violation: InvalidReferenceViolation [axiom=Declaration(ObjectProperty(<http://usefulinc.com/ns/doap#file-release>)), referencedObject=<http://usefulinc.com/ns/doap#file-release>, category=DANGLING]`

**Comment by jamesaoverton** [comment]
I think this message is just saying that `owl:Thing` is not defined, so when `BFO:0000001` refers to `owl:Thing` the reference is "dangling". That's safe to ignore. So I don't want to see these messages. @cmungall ?

**Comment by cmungall** [comment]
Yes, owl:Thing does not conform to OBO principles! I think including all owl builtins (top/bottom x class/property) should be ignored
